### ISSUE TITLE
Release version 1.0, adhere to semantic versioning?

### ISSUE THREAD

**Issue by tkoolen** [JuliaGizmos/WebIO.jl#222]
I noticed that the diff between 0.4.0 and 0.4.1 has 107 changed files, containing 35 commits with both new features and breaking API changes. So in semantic versioning terms, this should probably have been a new major version, as opposed to a patch version.

Although semver allows for this with major version zero (https://semver.org/#spec-item-4), I think it may be time to consider the initial development phase over given the amount of code that depends on WebIO. Tagging version 1.0.0 and adhering strictly to semver from then on would really help communicate the magnitude of introduced changes to downstream package maintainers.

**Comment by SimonDanisch** [comment]
Sounds good to me :) Would be nice to fix https://github.com/JuliaGizmos/WebIO.jl/issues/221 first though :P

**Comment by tkoolen** [comment]
Yeah, also #219 (my primary motivation for this issue).

**Comment by shashi** [comment]
Yup we need to patch both those issues... Sorry about #219, we can look at it if you're coming to CAJUN tomorrow and are interested.

Also, I see only 5 commits in https://github.com/JuliaGizmos/WebIO.jl/compare/v0.4.0...v0.4.1 , 2 bug fixes.

**Comment by tkoolen** [comment]
My, mistake, you're right. I was looking at the wrong thing.
